Tools/scripts/check_extension_modules.py

```diff
@@ -0,0 +1,487 @@
+"""Check extension modules
+
+The script checks extension and built-in modules. It verifies that the
```

Comment:
I'm confused by the first sentence. The two kinds of extensions are: shared (libraries) and built-in, no?

```suggestion
The script checks shared and built-in extension modules. It verifies that the
```